### PR TITLE
OEL-2806: Use the same warning message for user subscriptions request token email.

### PR DESCRIPTION
**Pull request by AaronGilMartinez**
…t message.

### CODE REVIEW

File: modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module [hunk 1]
```diff
@@ -61,7 +61,10 @@ function oe_subscriptions_anonymous_mail($key, &$message, $params) {
 function oe_subscriptions_anonymous_theme($existing, $type, $theme, $path) {
   return [
     'oe_subscriptions_anonymous_message_confirm' => [
-      'variables' => [],
+      'variables' => [
```
Comment (brummbar) [inline modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module:64]: The work is good, but I would:

1. Not default any values, require title/message to be passed always.
2. Change the template hook name: maybe we could name it `oe_subscriptions_anonymous_status_message` ?

**Review by brummbar — CHANGES_REQUESTED** [review]

File: modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module (hunk 1 above)
Comment (AaronGilMartinez) [reply modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module:64]: The name change is Ok. 
About the require of the title/message, removed default values, and to be sure, you want to make them mandatory? Like no displaying anything or throwing an error?

Comment (brummbar) [reply modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module:64]: Often Drupal uses NULL and doesn't really check anything. Let's check for each value and do not print anything if not there.

Comment (AaronGilMartinez) [reply modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module:64]: Changed to NULL, added checks

**Review by brummbar — APPROVED** [review]

**Pull request merged**